### DISCUSSION TITLE
Atributos de Usuário

### DISCUSSION THREAD

**Discussion by zaqmarlon** [isaacmsl/ezcommerce#36] [Ideas]
O frontend propõe os seguintes atributos:

- Interesse (Apenas comprar, Apenas vender, Comprar ou vender)
- Perfil do usuário (Empresa de pequeno/médio/grande porte ou Autônomo)
- Nome completo
- Email
- Senha

Baseado em:

![image](https://user-images.githubusercontent.com/31693006/111245224-11f0db80-85e3-11eb-9126-7eb39fc01421.png)

**Comment by PauloVLB** [comment]
Em que seriam usados os atributos Interesse e Perfil? Eles mudariam o comportamento do usuário em algum momento?

**Reply by zaqmarlon** [reply]
A exemplo da usuária Rebeca, ela deseja que seus produtos sejam marcados como confiáveis, por se tratar de uma empresa. O front poderia colocar um ícone de "Produto verificado" nos produtos desses "tipos" de usuário. O campo "Interesse" seria apenas um dado que seria usado no futuro para a evolução do sistema.

**Reply by PauloVLB** [reply]
Então, no momento, a única restrição para que um produto fosse "Verificado" seria o tipo de usuário que o criou? Não acho que faça sentido.

**Reply by Doug3321** [reply]
Concordo com Paulinho que esse perfil eu não acho que faça muito sentido, mas eu gostei da ideia do interesse

**Reply by zaqmarlon** [reply]
É, por enquanto seria só pelo tipo, mas se fosse evoluir, nossa equipe deveria verificar a existência dessa empresa, verificar qualidade do produto etc. Até lá, esses dois campos não passam de dados de "reconhecimento" dos usuários, com eles podemos criar análises.

**Reply by zaqmarlon** [reply]
"60%" dos nossos usuários declaram terem empresas de médio porte
"30%" são autônomos
"75%" declaram só utilizarem a plataforma para comprar produtos
etc

**Reply by PauloVLB** [reply]
Então, para podermos usá-los, teríamos que criar o caso de uso "gerar estatísticas do sistema".

**Reply by zaqmarlon** [reply]
Na evolução do projeto poderíamos sim criar esse caso de uso (bem mais bem verde mesmo kkkk)

**Reply by zaqmarlon** [reply]
Mas tipo, eu acho legal colocar antes porque garantimos um possível uso futuro. Já que os usuários se cadastram e já informam (seria ruim se só informassem depois), entende?

**Comment by zaqmarlon** [comment]
Foi decidido que os atributos "Interesse" e "Perfil" serão desconsiderados.
